main/src/ca/uwaterloo/flix/util/Duration.scala

```diff
@@ -43,7 +43,7 @@ object Duration {
 /**
   * A simple class to format a time duration.
   */
-class Duration(val d: Long) {
+case class Duration(val d: Long) {
```

Comment:
Redundant `val` :)

Reply:
Would be nice if this said what `d` represents (units)

Reply:
Done!